apps/files/models.py

```diff
@@ -378,24 +380,6 @@ def get_localepicker(self):
         statsd.timing('files.extract.localepicker', (end * 1000))
         return res
 
-    @property
-    def is_signed(self):
-        """Return True if the file has been signed.
-
-        This will simply check the signature filenames, and assume that if
-        they're named "mozilla.*" then the xpi has been signed by us.
-
-        This is in no way a perfect or correct solution, it's just the way we
-        do it until we decide to inspect/walk the certificates chain to
-        validate it comes from Mozilla."""
-        try:
-            with zipfile.ZipFile(self.file_path, mode='r') as zf:
-                filenames = set(zf.namelist())
-        except (zipfile.BadZipfile, IOError):
-            filenames = set()
-        return set(['META-INF/mozilla.rsa', 'META-INF/mozilla.sf',
-                    'META-INF/manifest.mf']).issubset(filenames)
```

Comment:
That's a good idea, but as soon as we have the signing activated on production, we'll never have this case anymore (a file that is not checked): when a new file is created, it'll be not signed, and then signed when we review it.

It would still be useful for the switch, and maybe even desirable for the migration (because adding a field with a default value will lock the whole table while it's being added). But we would never use it afterwards.

I could split the migration in two (first adding a nullable field, and then setting its default value and removing the nullable) if needed, what do you think @jasonthomas ?